### DISCUSSION TITLE
Billing question

### DISCUSSION THREAD

**Discussion by mrogozhin** [deepgram/community#400] [General help]
Hello. My project has been using Deepgram since June 1, 2023. We use Nova streaming model, have Pay As You Go plan and auto-reload turned on. 
In total, we spent 67260 minutes. Based on $0.0059/min pricing, we should have been charged around $400 in total.  
In fact, our total balance with all auto-reloads is $950 (including starters $200 promo balance and $100 which are still not spent). I see some discrepancy in numbers: **$400 VS $850** (not including not-spent-yet $100).
I wasn't able to find a way to contact Deepgram support directly, so I'm asking here - could somebody help to investigate this issue? 

![image](https://github.com/deepgram/community/assets/38915921/a0ae2894-4a10-49b2-bc5a-b7661cec9b78)

**Comment by jkroll-deepgram** [comment]
Hi @mrogozhin, can you please provide your project ID or a request ID, and we can look into the details of your account billing from there.

**Reply by mrogozhin** [reply]
Project ID 1ea52b4e-b121-4d78-83f9-030535cbdd4d

**Reply by jkroll-deepgram** [reply]
Thanks for the quick reply. I am looking into this and will get back to you asap within the next day or two.

**Reply by mrogozhin** [reply]
Thanks a lot! I'll be waiting for updates

**Reply by jkroll-deepgram** [reply]
I've reached out via email. We've identified an isolated billing error that we will resolve, and are making a fix so it does not reoccur.

**Reply by mrogozhin** [reply]
Thanks a lot for a quick help. I responded via email with my desired resolution.
